Basic/APIDiorama/.gitignore

```diff
@@ -0,0 +1,72 @@
+# This .gitignore file should be placed at the root of your Unity project directory
```

Comment:
any reason why this is project specific and not repo wide?

Reply:
Because I didn't want to mess up with the global gitignore. If you all agree ith this content, we can have it global. Let me know. cc @fernando-cortez @LPLafontaineB

Reply:
oh wasn't aware. any reason why LP?

Reply:
I do not know, this is from before I joined the team. I suppose changing this pattern simply never came up in a discussion before, but yeah I think it would make things cleaner instead of having to maintain multiples gitignore files that are functionally identical anyways

Reply:
I agree with unification, but let's address this in a separate PR to not block this one even further.